syntax = "proto3";
package protobufs_models;

import "google/protobuf/timestamp.proto";


message User{
    string id = 1;
    string name = 2;
    string phone = 3;
    string email = 4;
    google.protobuf.Timestamp created_at = 5; 
}